LEAF-USCS/lf-be
# Setup Back-end Environment

### What does this PR do?
* Setup the server side environment for the leaf platform

### Description of Task to be completed?
* Install Express server and all relevant node packages
* Setup Postgres database with Sequelize ORM
* Create test endpoint to test the server

### Any background context you want to provide?
* Make sure the sequelize-cli package is installed globally. 
* Run _npm install -g sequelize-cli_

N/A

### How To Manually Test 
* Pull down this branch and checkout to it
* Run _npm install_ to install all packages
* Ensure all environment variables are set and that your database is setup
* Run _npm start_ and make a request to _/api/test_

### Questions:
N/A
